### PR TITLE
Make tablesort working together with a tablesort indicator

### PR DESCRIPTION
**Pull request by dawehner**
Solves #1 , but also integrates a tablesort indicator

### CODE REVIEW

**Comment by mattgrill** [comment]
@mglaman Any feedback on this?

**Comment by mglaman** [comment]
This looks way better than mine. However I realize we do not proper pass the queries for ASC/DESC.

Must be `sort_by` and `sort_order` which fails if not ASC or DESC (case matters.)

File: js/src/components/app.js [hunk 1]
```diff
@@ -43,6 +44,7 @@ export default class App extends Component {
   sortHandler = (sort, order) => {
     this.setState({
       order: (order === 'desc' && 'asc') || 'desc',
+      sortBy: sort,
       filterParams: Object.assign(this.state.filterParams, { sort_by: `${sort}_${order}` }),
```
Comment (mattgrill) [inline js/src/components/app.js:48]: We set the new `sort_by` here which is `${sort}_${order}`.

**Comment by mattgrill** [comment]
@mglaman I didn't encounter this behavior, additionally I didn't see that case made any difference.

**Pull request merged**